### ISSUE TITLE
luci-mod-network: multi-port devices (>9) incorrectly sorted

### ISSUE THREAD

**Issue by oliv3r** [openwrt/luci#5899]
Any page featuring device ports will mess up the ordering (due to using digital sorting instead of natural sorting, e.g. 0, 1, 10, 11, 2, 3)

## Steps to reproduce:
Running luci master on a device with more then 9 ports

1. go to: Network → Interfaces → Devices

## Actual behavior:
![image](https://user-images.githubusercontent.com/1245341/181246345-00b2f190-b2f0-460f-ac5a-e13d09558ae4.png)

also on bridge ports:
![image](https://user-images.githubusercontent.com/1245341/181246627-59a8cb54-27d1-40cf-83fa-849c90b942b7.png)

and most importantly also on the vlan page!
![image](https://user-images.githubusercontent.com/1245341/181247996-f2d165ef-996b-400e-9c91-73182071957e.png)

However, dismissing the dialog, and re-opening it, will sort it properly! (this does not apply to the other cases listed above.
![image](https://user-images.githubusercontent.com/1245341/181246530-8abe68d4-660c-4ffc-869e-30e7a6fdc293.png)

## Expected behavior:
As a user, having the ports ordered logically, is far more user-friendly, and makes it cognitively easier to follow.

## Additional Information:
OpenWrt version information from system `/etc/openwrt_release`

```
DISTRIB_ID='OpenWrt'
DISTRIB_RELEASE='SNAPSHOT'
DISTRIB_REVISION='r20181+4-a436923133'
DISTRIB_TARGET='realtek/rtl930x'
DISTRIB_ARCH='mips_24kc'
DISTRIB_DESCRIPTION='OpenWrt SNAPSHOT r20181+4-a436923133'
DISTRIB_TAINTS='no-all busybox'
```

**Comment by jow-** [comment]
Should be fixed with 8199b2ce9a, 37422e891a, 801d4513b2

**Comment by oliv3r** [comment]
Confirmed closed: while you already merged; but you can have my tested by :)
